hubble-ui/BRANCH

```diff
@@ -1 +1 @@
-0.10
+0.12
```

Comment:
The default hubble-ui version in the Cilium v1.13.7 is 0.11. I would suggest using hubble-ui v0.11 for now.
https://github.com/cilium/cilium/blob/1.13.7/install/kubernetes/cilium/values.yaml#L1225

Reply:
Thank you for your review.

In this document, hubble v0.11 is not maintained now.
And hubble v0.12 supports cilium v0.14 and older version.
So I think we should use hubble v0.12 and the same version of hubble-ui.

https://github.com/cilium/hubble#releases

Reply:
This document is about hubble cli, not hubble-ui, right? As far as I know, the versions of hubbl cli and hubble-ui do not have to the same.

The reason I suggest it is that I used the latest hubble-ui that Cilium hasn't supported and that broke it due to the Nginx config change.

Reply:
FYI, nginx.conf for hubble-ui is here
https://github.com/cilium/cilium/tree/main/install/kubernetes/cilium/templates/hubble-ui